### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by mohashyne**
# New  Features Added

With this feature, I worked on the  following:
- [ ] improve the look and feel of the application, by adding styles to enhance the HTML structure.
- [ ]  Improved  the layout
- [ ] updated readme file

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -70,8 +98,11 @@ input[type="text"] {
...
 
 #submit-btn {
+  width: 100px;
   padding: 5px;
   border: 1px solid #ccc;
   background-color: #fff;
   cursor: pointer;
+  border-radius: 3px;
+  box-shadow: 0 0 5px #ccc;
 }
```
Comment (usorfaitheloho) [inline src/styles/main.scss:122]: Hi @mohashyne, you did well with your leaderboard project 👍 . Please remember that you are to copy the design as closely as possible so please implement these changes :

1.  the `Add your score` should be a heading and it should be centralized,as it's a heading of a subsection
2.  The `Submit` button should be on the right, just as it is in the design given
3. The `recent scores` and the `Refresh`  shouldn't be together. the first is on the left while the other is on the right of the same subsection

Your design :
leaderboard-image

Framework :

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @mohashyne  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/main.scss (hunk 1 above)
Comment (mohashyne) [reply src/styles/main.scss:122]: Requested changes applied

**Review by jerryowusu — APPROVED** [review]
Hi @mohashyne 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

CookedDelectableDolphin-size_restricted


## Highlights
Great UI :+1: 
Great effort in making changes :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**